lib/puppet/pops/lookup/hiera_config.rb

```diff
@@ -173,6 +175,11 @@ def initialize(config_root, config_path, loaded_config)
     @data_providers = nil
   end
 
+  def fail(issue, args = EMPTY_HASH, line = nil)
```

Comment:
Ping @thallgren - do you agree? Change `fail` here to something else, and then make the corresponding change elsewhere at some other point?